examples/00_10_cell_pull_face.py

```diff
@@ -3,11 +3,8 @@
 from __future__ import division
 
 import compas
-
-from compas_rhino.helpers import mesh_from_surface
-
+from compas_rhino.geometry._constructors import mesh_from_surface
```

Comment:
replace by `from compas.geometry import RhinoSurface`
use like this

`mesh = RhinoSurface.from_guid(guid).brep_to_compas(cls=Cell)`